src/librustc_trait_selection/traits/query/normalize.rs

```diff
@@ -128,8 +128,7 @@ impl<'cx, 'tcx> TypeFolder<'tcx> for QueryNormalizer<'cx, 'tcx> {
             }
 
             ty::Projection(ref data) if !data.has_escaping_bound_vars() => {
-                // (*)
-                // (*) This is kind of hacky -- we need to be able to
+                // This is kind of hacky -- we need to be able to
```

Comment:
Please avoid changing comments, this is some preexisting notation.

Reply:
so the (*) is not used as some kind of link/reference and was forgotten during a refactor? :scream:

Reply:
It's weirder than that, you'll have to ask @nikomatsakis

Reply:
aah, its meant for path match arms.